TACC/protx
# Task/cooks 74

## Overview: ##

Replaced the geoid code for the selected area type with the FIPS Name.

## Related Jira tickets: ##

* [COOKS-74](https://jira.tacc.utexas.edu/browse/COOKS-74)

## Summary of Changes: ##

## Testing Steps: ##

1. Run project.
2. Select region on Mal;treatment map.
3. Ensure Selected value reflects county name.

## UI Photos:

<img width="1639" alt="Screen Shot 2021-08-13 at 7 44 03 PM" src="https://user-images.githubusercontent.com/3238078/129429509-26031ad8-8c37-4db9-a145-92ab7af86e54.png">


## Notes: ##
